backend/tests/integration/test_file_upload_validation.py

```
@@ -0,0 +1,505 @@
+"""
+Integration tests for file upload validation.
+Tests the complete upload endpoint with various file scenarios.
+"""
+
+import pytest
```

Ruff (0.11.9)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)